packages/fluentui/react-builder/src/components/Canvas.tsx

```diff
@@ -0,0 +1,311 @@
+              element.removeAttribute('data-builder-id');
+              const { width, height } = element.getBoundingClientRect();
+              const { marginTop, marginRight, marginBottom, marginLeft } = iframeWindow.getComputedStyle(element);
+              element.setAttribute('data-builder-id', builderId);
+
+              const hasNoWidth = width === 0;
+              const hasNoHeight = height === 0;
+              const hasNoChildren = element.childElementCount === 0;
+              const hasManyChildren = element.childElementCount > 1;
+
+              const properties = [
+                hasNoChildren &&
+                  hasNoWidth &&
+                  `padding-left: calc(${debugSize} * 2);\n  padding-right: calc(${debugSize} * 2);`,
+                hasNoChildren &&
+                  hasNoHeight &&
+                  `padding-top: calc(${debugSize} * 2);\n  padding-bottom: calc(${debugSize} * 2);`,
+                hasManyChildren && `padding: ${debugSize};`,
+                marginTop === '0px' && `margin-top: ${debugSize};`,
+                marginRight === '0px' && `margin-right: ${debugSize};`,
+                marginBottom === '0px' && `margin-bottom: ${debugSize};`,
+                marginLeft === '0px' && `margin-left: ${debugSize};`,
+              ]
+                .filter(Boolean)
+                .join('\n');
+
+              // console.log(
+              //   element,
+              //   '\nHAS\n',
+              //   { width, height, marginTop, marginRight, marginBottom, marginLeft },
+              //   '\nGETS\n',
+              //   properties,
+              // );
+
+              return properties.length === 0 ? '' : `[data-builder-id="${builderId}"] {\n${properties}\n}`;
+            })
+            .filter(Boolean)
+            .join('\n');
+
+      style.innerHTML = [
+        isSelecting &&
+          `
+          [data-builder-id="builder-root"] {
+            ${isExpanding ? `padding: ${debugSize};` : ''}
+            min-height: 100vh;
+          }
+          `,
+        isExpanding &&
+          `
+          [data-builder-id]:not([data-builder-id="builder-root"]) {
+            outline: 1px dotted cornflowerblue;
+            outline-offset: -1px;
+          }
+          `,
+        elementStyles,
+      ]
+        .filter(Boolean)
+        .join('\n');
+
+      // console.log('Canvas:effect style element:', style.innerHTML);
+    });
+
+    return () => {
+      const iframe = document.getElementById(iframeId) as HTMLIFrameElement;
+
+      // console.log('Canvas:effect clean up');
+
+      iframe.contentWindow.clearTimeout(animationFrame);
+    };
+  }, [iframeId, isExpanding, isSelecting]);
+
+  return (
+    <Frame
+      title="Designer Canvas"
+      frameBorder="0"
+      width="100%"
+      height="100%"
+      initialContent='<!DOCTYPE html><html><head><style>html {font-size: 14px;}</style></head><body><div class="frame-root"></div></body></html>'
```

Comment:
```suggestion
      initialContent='<!DOCTYPE html><html><head><style>html {font-size: VALUE;}</style></head><body><div class="frame-root"></div></body></html>'
```

Should we take `fontSize` value from the main window?